roles/openshift_daemonset_config/templates/daemonset.yml.j2

```diff
@@ -0,0 +1,123 @@
+---
```

Comment:
Yedit instead of parameterization?  This is almost not at all parameterized, so a good candidate